### ISSUE TITLE
does not work when started via flask run

### ISSUE THREAD

**Issue by ror6ax** [opentracing-contrib/python-flask#5]
The following script never starts the webserver and just hangs forever:

```
import opentracing
from flask import Flask
from flask_opentracing import FlaskTracer
from jaeger_client import Config

app = Flask(__name__)

config = Config(
config={ # usually read from some yaml config
                          'sampler': {
                          'type': 'const',
                          'param': 1,
                  },
                          'logging': True,
                  },
service_name='flask_app_1',
)


opentracing_tracer = config.initialize_tracer()
tracer = FlaskTracer(opentracing_tracer)

@app.route('/some_url')
@tracer.trace()
def some_view_func():
    return "some_view"
```

With debug logging turned on, I see this:

`2017-11-06 15:10:16,177 Initializing Jaeger Tracer with UDP reporter`

Same script with minimal changes started by python scriptname.py works as a charm.

**Comment by ror6ax** [comment]
Hi guys, any update on this?

**Comment by ror6ax** [comment]
Addressed by #4
